assets/prototype/domain/eventEditor/datetimes/dateList/filterBar/useDatesListFilterState.js

```diff
@@ -35,6 +36,7 @@ const useDatesListFilterState = () => {
 	return {
 		datesSortedBy: state.datesSortedBy,
 		displayDates: state.displayDates,
+		filteredDates: state.filteredDates,
 		setDatesSortedBy,
 		setDisplayDates,
 		setShowDates,
```

Comment:
why not simply this?
```js
{
	...state,
	setDatesSortedBy,
	setDisplayDates,
	setShowDates,
}
```
If it's for readability, then we can define the type for it.

Reply:
Yes, that is simpler, updated: https://github.com/eventespresso/event-espresso-core/pull/2087/commits/cbdefec12fd440721229e0eb12560c5e17630eb2

Reply:
Also added a comment in other related issue to add more types: https://github.com/eventespresso/event-espresso-core/issues/2091#issuecomment-568756040